tests/perf-system/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,513 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import json
+import os
+import random
+import signal
+import stat
+import string
+import subprocess
+import sys
+import time
+from pathlib import Path
+from loguru import logger as LOG
+
+import paramiko
+
+
+def connect_to_remote_server(ip_address):
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    client.connect(ip_address)
```

Comment:
Having an open ended function, as opposed to a context manager (eg. https://github.com/microsoft/CCF/blob/9e0e46c41a306d10dc128f3f3672ad16a58cc089/tests/infra/remote.py#L43) runs the risk that the connection will stay open for longer than intended if the script otherwise raises an uncaught exception.

Reply:
This particular instance has been replaced with a context manager in ab101aa0ac216184d0a993058863ec7ab2e703cd